arm-cross-compile.cmake

```diff
@@ -0,0 +1,29 @@
+set(CMAKE_SYSTEM_NAME Generic)
```

Comment:
Does this file belong in the `./cmake/` directory?

Do we need another that lets you build an tinylibiio backend for linux? (no threads, with it's own back end, to show people how to incorporate everything into a single binary?

Reply:
Yes, `./cmake/` is a much better place.
It would be nice to have an example that showcases how to define a custom backend. But I would add that in a separate PR.